### PR TITLE
Client debug property

### LINKED ISSUE #1012: Client debug mode should be tied to java debug mode
We have two configuration properties to set debug mode:

- `bordertech.wcomponents.debug.enabled` and 
- `bordertech.wcomponents.debug.clientSide.enabled`.

To enable client side debugging both of these must be set true. This is completely superfluous and client debuggon should be on by default when WComponents debug mode is on.

### PR DESCRIPTION
**Pull request by marksreeves**
Ensure that debug mode is consistent for server and client code. Found
and fixed a HTML error in DevToolkit_header.vm.

Resolves #1012.

@jonathanaustin PTAL

### CODE REVIEW

File: wcomponents-lde/src/main/java/com/github/bordertech/wcomponents/lde/PlainLauncher.java [hunk 1]
```diff
@@ -109,8 +103,6 @@ public synchronized WComponent getUI(final Object httpServletRequest) {
 	@Override
```
Comment (jonathanaustin) [inline wcomponents-lde/src/main/java/com/github/bordertech/wcomponents/lde/PlainLauncher.java]: This method could be removed as it only calls super.

File: wcomponents-lde/src/main/java/com/github/bordertech/wcomponents/lde/PlainLauncher.java [hunk 2]
```diff
@@ -124,19 +116,7 @@ public InterceptorComponent createInterceptorChain(final Object request) {
 		// The toolkit must render itself within the page shell
 		// and wrap the main WComponent output.
 		InterceptorComponent.replaceInterceptor(PageShellInterceptor.class,
```
Comment (jonathanaustin) [inline wcomponents-lde/src/main/java/com/github/bordertech/wcomponents/lde/PlainLauncher.java]: As this is now replacing itself with itself, this override method is no longer needed.

File: wcomponents-lde/src/main/java/com/github/bordertech/wcomponents/lde/PlainLauncher.java (hunk 1 above)
Comment (marksreeves) [inline wcomponents-lde/src/main/java/com/github/bordertech/wcomponents/lde/PlainLauncher.java]: 👍

File: wcomponents-lde/src/main/java/com/github/bordertech/wcomponents/lde/PlainLauncher.java (hunk 2 above)
Comment (marksreeves) [inline wcomponents-lde/src/main/java/com/github/bordertech/wcomponents/lde/PlainLauncher.java]: 👍

**Comment by marksreeves** [comment]
@jonathanaustin all items in review completed. Thanks.

**Pull request merged**